.github/workflows/build-and-test-orchard-core.yml

```diff
@@ -76,6 +76,11 @@ jobs:
     - uses: actions/checkout@v3
       with:
         submodules: 'recursive'
+    # Firefox and Chrome browsers are installed by default on ubuntu-latest and windows-latest, and Edge is installed by
+    # default on windows-latest, so here we install Edge on ubuntu-latest. It seems Linux support is planned here
+    # (https://github.com/browser-actions/setup-edge), after this completes we can use it.
+    - name: Set up Edge
```

Comment:
Since currently only the OSOCE build is where we need this, make it optional and disabled by default, just enable it from the OSOCE workflow and just for non-NuGet build.